src/pages/messaging/customer-chat-api/index.mdx

```diff
@@ -795,6 +795,7 @@ The method updates the requester's `events_seen_up_to` as if they've seen all ch
 curl -X POST \
   'https://api.livechatinc.com/v3.5/customer/action/send_event?organization_id=<organization_id>' \
   -H 'Content-Type: application/json' \
+  -H 'Authorization: Bearer <customer_access_token>' \
```

Comment:
@litwinow I would appreciate it if you could just confirm if this is correct now🙏🏻 We have an issue reported on [#platform-docs](https://text.slack.com/archives/C2XMTMQ5S/p1678181079893809) that the authorization block is missing in the `Send Event` method. I added it in v3.4-3.6 as the earlier ones have it. In my opinion it is okay now, but I would like to double check

Reply:
confirming that it's now ok